### PR TITLE
Tree popin page search #534

### PR DESCRIPTION
**Pull request by oanesbogdan**
Search page in tree popin.
https://github.com/backbee/backbee-standard/issues/534

### CODE REVIEW

File: src/tb/i18n/en_US/global.json [hunk 1]
```diff
@@ -254,6 +254,7 @@
     "scheduling": "Scheduling",
     "search": "Search",
     "search_content": "Search content",
+    "search_page": "Search a page",
```
Comment (fkroockmann) [inline src/tb/i18n/en_US/global.json]: Can you remove this key and use search key plz ?

**Comment by fkroockmann** [comment]
Very good work, can you rebase plz ?

**Comment by oanesbogdan** [comment]
I made the modifications. Thanks!

**Comment by fkroockmann** [comment]
Thanks @oanesbogdan

**Pull request merged**